core/src/main/resources/template/simple/optgroup.ftl

```diff
@@ -35,10 +35,37 @@
 	<#assign tmpKey=stack.findValue(optGroupInternalListUiBean.parameters.listKey) />
 	<#assign tmpValue=stack.findValue(optGroupInternalListUiBean.parameters.listValue) />
 	<#assign tmpKeyStr = tmpKey.toString() />
+	<#assign optGroupItemCssClass = ''/>
+	<#if optGroupInternalListUiBean.parameters.listCssClass??>
+		<#if stack.findString(optGroupInternalListUiBean.parameters.listCssClass)??>
+			<#assign optGroupItemCssClass= stack.findString(optGroupInternalListUiBean.parameters.listCssClass)/>
+		</#if>
+	</#if>
+	<#assign optGroupItemCssStyle = ''/>
+	<#if optGroupInternalListUiBean.parameters.listCssStyle??>
+		<#if stack.findString(optGroupInternalListUiBean.parameters.listCssStyle)??>
+			<#assign optGroupItemCssStyle= stack.findString(optGroupInternalListUiBean.parameters.listCssStyle)/>
+		</#if>
+	</#if>
+	<#assign optGroupItemTitle = ''/>
+	<#if optGroupInternalListUiBean.parameters.listTitle??>
+		<#if stack.findString(optGroupInternalListUiBean.parameters.listTitle)??>
+			<#assign optGroupItemTitle= stack.findString(optGroupInternalListUiBean.parameters.listTitle)/>
+		</#if>
+	</#if>
 	<option value="${tmpKeyStr?html}"<#rt>
 	<#if tag.contains(parameters.nameValue, tmpKey) == true>
 	selected="selected"<#rt>
 	</#if>
+	<#if optGroupItemCssClass?has_content>
```

Comment:
hm... `?has_content` is pretty much the same as `??` but it supports collections, do we expect collections here? Or did I miss something?

Reply:
It will be assigned empty string in case there is no attribute in the tag and `??` doens't check for empty strings.

Reply:
👍